src/ORM/Behavior/TreeBehavior.php

```diff
@@ -557,51 +557,58 @@ public function moveUp(Entity $node, $number = 1)
      */
     protected function _moveUp($node, $number)
     {
-        $config = $this->config();
-        list($parent, $left, $right) = [$config['parent'], $config['left'], $config['right']];
-
         if (!$number) {
             return false;
         }
-
-        $parentLeft = 0;
-        if ($node->get($parent)) {
-            $parentLeft = $this->_getNode($node->get($parent))->get($left);
+        if ($number < 0) {
```

Comment:
I think the intent would have been to catch negatives as well. Negative values make no sense in MPTT.